avocado/utils/configure_network.py

```diff
@@ -0,0 +1,66 @@
+"""
+Configure network when interface name and
+interface IP is available.
+"""
+import shutil
+
+from . import distro
+from . import process
+
+
+def set_ip(ipaddr, netmask, interface):
+    """
+    Gets interface name, IP, subnetmask and creates interface
+    file based on distro.
+    """
+    if distro.detect().name == 'rhel':
+        conf_file = "/etc/sysconfig/network-scripts/ifcfg-%s" % interface
+        shutil.move(conf_file, conf_file+".backup")
```

Comment:
@vaishnavibhat ah sorry i did not notice this earlier , one issue if  `interface` not available or by mistake user provide a wrong `interface` name 

utility will error out as file move failed as FileNotFoundError: [Errno 2
as example as 
`FileNotFoundError: [Errno 2] No such file or directory: '/etc/sysconfig/network-scripts/ifcfg-sd'` 

please check that if interface file available in sysconfig path 

can be like
``` 
if os.path.exists(conf_file):
  shutil.move(conf_file, conf_file+".backup")
else 
   raise OSError('% interface not available " % interface) 
```